.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
```

Comment:
This might need tweaking to be less annoying (filtering on semver:major only maybe), we'll see. I'll do it as needed.

Reply:
Filtering to semver:major does seem pretty reasonable.  But also...maybe weekly isn't too bad regardless.  Is there a dependabot or whatever that can open PRs automatically to take the weekly pain down a bit?

Reply:
this will open PRs yes. https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates#about-dependabot-pull-requests